core/modules/image/image.field.inc

```diff
@@ -617,22 +624,29 @@ function _image_field_widget_alt_validate($element, &$form_state) {
  * Implements hook_field_formatter_info().
  */
 function image_field_formatter_info() {
-  $formatters = array(
-    'image' => array(
-      'label' => t('Image'),
-      'field types' => array('image'),
-      'settings' => array(
-        'image_style' => '',
-        'image_link' => '',
-        'image_float' => '',
-        'image_load' => 'auto',
-      ),
-	  'description' => t('Format the file as an image. The image can be displayed using an image style and can optionally be linked to the image file itself or its parent content.'),
-	  'file formatter' => array(
-        'mime types' => array('image/*'),
-      ),
+  $image_formatter = array(
+    'label' => t('Image'),
+    'field types' => array('image'),
+    'settings' => array(
+      'image_style' => '',
+      'image_link' => '',
+      'image_float' => '',
+      'image_load' => 'auto',
+      'image_svg_settings' => array(),
+    ),
+    'description' => t('Format the file as an image. The image can be displayed using an image style and can optionally be linked to the image file itself or its parent content.'),
+    'file formatter' => array(
+      'mime types' => array('image/*'),
     ),
   );
+  $formatters['image'] = $image_formatter;
+
+  $image_formatter['label'] = t('Image (SVG Support)');
```

Comment:
I understand what's going on here, however the choice of the variable name here made me pause for a minute and go re-read through the code. Can we please start with a new `$svg_formatter` variable instead of adding keys to the previously defined `$image_formatter` (and adjust the surrounding code accordingly)?

Reply:
I'm removing the `Image (SVG Support)` formatter completely. This was added by the SVG Image module, but since we're merging into core, we just need one image formatter.